src/palindrom.js

```diff
@@ -823,6 +765,10 @@ var Palindrom = (function () {
 
   /* backward compatibility */
   global.Puppet = Palindrom;
+  
+  /* Since we have jsonpatch bundled,
+  let's expose it in case anyone needs it */
+  global.jsonpatch = jsonpatch;
```

Comment:
Is there currently a need for exposing `jsonpatch`? I have mixed feelings about making it public. It is quite unexpected to get dependencies exposed in the global namespace. Exposing it as `Palindrom.jsonpatch` would be better perhaps. @tomalec WDYT?